### PR TITLE
 Use scrollIntoView() for FIREFOX and EDGE browsers before clicking

### PR DESCRIPTION
**Pull request by AlenaBudevich**

### CODE REVIEW

File: vividus-plugin-web-app/src/test/java/org/vividus/ui/web/action/MouseActionsTests.java [hunk 1]
```diff
@@ -497,12 +497,13 @@ void testMoveToElementOnMobile()
     }
 
     @Test
-    void testMoveToElementOnSafari()
+    void testMoveToElementOnAnyTypeOf()
```
Comment (valfirst) [inline vividus-plugin-web-app/src/test/java/org/vividus/ui/web/action/MouseActionsTests.java]: `testMoveToElementOnAnyTypeOf` -> `shouldScrollToElementOnBrowsersNotPerformingScrollAutomatically`

File: vividus-plugin-web-app/src/main/java/org/vividus/ui/web/action/MouseActions.java [hunk 2]
```diff
@@ -183,8 +183,9 @@ public void moveToElement(WebElement element)
     {
         if (element != null)
         {
-            //Safari Driver doesn't scroll to element before moveTo action
-            if (webDriverManager.isMobile() || webDriverManager.isTypeAnyOf(WebDriverType.SAFARI))
+            //Safari, Firefox and Edge Drivers don't scroll to element before moveTo action
```
Comment (uarlouski) [inline vividus-plugin-web-app/src/main/java/org/vividus/ui/web/action/MouseActions.java]: // Safar

Comment (valfirst) [reply vividus-plugin-web-app/src/main/java/org/vividus/ui/web/action/MouseActions.java]: https://github.com/checkstyle/checkstyle/issues/6593

Comment (ikalinin1) [reply vividus-plugin-web-app/src/main/java/org/vividus/ui/web/action/MouseActions.java]: Still not clear for me, it just about setting some formal formatting, isn't it? :)

**Pull request merged**